### PR TITLE
Liquid Jet setup

### PR DESCRIPTION
**Pull request by vespos**
## Description
Add axis and function for the liquid jet setup at MFX

## Motivation and Context
New(-ish) device that we use at MFX

## How Has This Been Tested?
Used in a couple beamtimes already

## Where Has This Been Documented?
N/A

## Pre-merge checklist
- [x] Code works interactively
- [x] Code contains descriptive comments
- [x] Test suite passes locally
- [x] Libraries are set to fixed versions and not ``Always Newest``
- [x] Code committed with ``pre-commit`` (alternatively ``pre-commit run --all-files``)

### CODE REVIEW

**Comment by vespos** [comment]
Trying to make that repo clean before we start the work on the MFX spectrometer.
@ZLLentz @klauer Can I please ask you to help me with the two failed checks here?

**Comment by klauer** [comment]
Fixing the pre-commit issue should be as simple as:
```
$ pre-commit install
$ pre-commit run --all-files
```

You should also do a `pre-commit install` on the programming node so you don't have to keep running the step separately - it will perform fixes as you commit new bits of code.

**Comment by klauer** [comment]
Additionally, for this repository, would you please do the following? (I noticed this while looking through all our PLC repositories.)

* Upgrade ads-ioc to R0.6.0 and rebuild
* `load_plc_databases.cmd` can then be removed as R0.6.0 does not use it
* Fix `PREFIX` in the IOC Makefile - it should perhaps be `PLC:MFX:MOTION` and not `PLC:mfx_motion` as it is currently

**Comment by vespos** [comment]
@klauer  I think comments were now addressed. Let me know if there is anything else?

File: iocBoot/ioc-mfx-motion/Makefile [hunk 1]
```diff
@@ -1,18 +1,25 @@
...
+PRODUCTION_IOC := 0
 
 PROJECT_NAME := plc-mfx-motion
 PROJECT_PATH := ../../plc-mfx-motion/plc-mfx-motion.tsproj
 PLC := mfx_motion
 
-PYTMC_OPTS := 
-PREFIX := PLC:$(PLC)
+PYTMC_OPTS :=
+PREFIX := PLC:MFX_MOTION
```
Comment (klauer) [inline iocBoot/ioc-mfx-motion/Makefile:12]: I think most other repositories have a form like `PLC:MFX:MOTION` (note the `_` -> `:` change)

Checking the gateway pvlists looks like mfx only cares about "MFX" in the name:
```
gateway-setup klauer$ git grep "PLC:"
config/kfe.pvlist:PLC:KFE:VAC:.*                ALLOW RWINSTRMCC 1
config/kfe.pvlist:PLC:KFE:.*            ALLOW RWINSTRMCC 1
config/kfe.pvlist:PLC:TMO:.*            ALLOW RWINSTRMCC 1
config/kfe.pvlist:PLC:RIX:.*            ALLOW RWINSTRMCC 1
config/rix.pvlist:PLC:.*                     ALLOW RWSXR 1
config/tmo.pvlist:PLC:.*                ALLOW
config/tmo.pvlist:PLC:TMO:.*            ALLOW RWINSTR 1

$ grep "MFX" config/mfx.pvlist
.*MFX.*         ALLOW
...
```

In that case I guess this is technically fine, but I'd still recommend switching it while we're here (sorry!)

**Review by klauer — APPROVED** [review]
Thanks for addressing the issues, @vespos!

File: iocBoot/ioc-mfx-motion/Makefile [hunk 2]
```diff
@@ -1,18 +1,25 @@
-IOC_TOP=/reg/g/pcds/epics/ioc/common/ads-ioc/R0.5.0
+IOC_TOP=/reg/g/pcds/epics/ioc/common/ads-ioc/R0.6.0
 IOC_INSTANCE_PATH := $(shell pwd)
 
 # Set PRODUCTION_IOC to 1 to move from a testing to a production IOC:
-PRODUCTION_IOC := 1
+PRODUCTION_IOC := 0
```
Comment (klauer) [inline iocBoot/ioc-mfx-motion/Makefile:5]:
```suggestion
PRODUCTION_IOC := 1
```

And don't forget to flip this back

**Pull request merged**